src/Ledger/Chain/Properties/CHAINepochStep.lagda

```diff
@@ -0,0 +1,50 @@
+\begin{code}[hide]
+{-# OPTIONS --safe #-}
+
+open import Ledger.Prelude
+open import Ledger.Abstract
+open import Ledger.Transaction
+import Ledger.Certs
+
+module Ledger.Chain.Properties.CHAINepochStep
+  (txs : _) (open TransactionStructure txs) (open Ledger.Certs govStructure)
+  (abs : AbstractFunctions txs)
+  where
+\end{code}
+
+\begin{property}[%
+  \LedgerMod{Chain/Properties/CHAINepochStep.lagda}{\AgdaModule{CHAINepochStep}}:
+  Chain state epoch step;
+  \textbf{to prove}%
+  ]\
+
+  \begin{itemize}
+    \item \textit{Informally}.
+      Let
+
+      \(\begin{array}{rcl}
+        \ab{s}&=&\text{a \AgdaRecord{ChainState}},\\
+        \ab{nes}&=&\text{the \AgdaField{newEpochState} of \ab{s}},\\
+        \ab{es}&=&\text{the \AgdaField{epochState} of \ab{nes} (= the \AgdaField{epochState} of \ab{s})},\\
+        \ab{le}&=&\text{the \AgdaField{lastEpoch} of \ab{nes}}.
+      \end{array}\)
```

Comment:
I like it more in the form of text as in the other properties than a list of bullet points.